playbooks/common/openshift-master/config.yml

```diff
@@ -5,6 +5,10 @@
     t_oo_option_master_debug_level: "{{ lookup('oo_option', 'openshift_master_debug_level') }}"
 
   pre_tasks:
+  - name: Fail early if Atomic and old version of Docker
+    include: ../../../roles/openshift_common/tasks/atomic_docker_check.yml
+    when: openshift.common.is_atomic | bool
```

Comment:
I'm not sure if we can rely on openshift.common.is_atomic being set at this point, since that is set in openshift_facts.